### PR TITLE
Add docs for process types

### PR DESCRIPTION
**Pull request by andreslucena**
Trying out a deployment for #130 after the introduction of the playbook changer logic.

If everything goes well, then the deployment preview should work 🤞🏽

### CODE REVIEW

**Comment by RCheesley** [comment]
@andreslucena I cannot seem to get the relative link to work, linking back to the assembly types anchor. I've tried a few different routes but just can't seem to figure out the syntax, even with the [docs](https://docs.antora.org/antora/latest/page/page-links/#modules).

I think that's the only thing blocking this now from what I can see. If someone can fix that then the PR is ready to go.

File: docs/en/modules/admin/pages/spaces/processes/types.adoc [hunk 1]
```diff
@@ -0,0 +1,38 @@
+= Process types
+
+In some cases it is necessary to organize participatory processes by more than just xref:spaces/processes/groups.adoc[Process Groups]. This is where Process Types can be useful.
+
+Process Types work similarly to xref:assemblies:assemblies.adoc#_assemblies_types[Assembly types].
```
Comment (andreslucena) [inline docs/en/modules/admin/pages/spaces/processes/types.adoc]: The xref are not intuitive for newcomers, at least for me were difficult to grok at the beginning. Most of the times [I go directly to the `nav.adoc` to copy and paste the reference](https://github.com/decidim/documentation/blob/8aba42fb91d31d7e18225ecfd6ba3524f907e904/docs/en/modules/ROOT/nav.adoc?plain=1#L68)

```suggestion
Process Types work similarly to xref:admin:spaces/assemblies.adoc#_assemblies_types[Assembly types].
```

This does the trick

File: docs/en/modules/admin/pages/spaces/processes/types.adoc [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+image:spaces/processes/process-types-admin-backend.png[A screenshot of the back-end showing the button to create a new process type,title="Create a new process type in the back-end"]
+
+.New participatory process type form: General information
+|===
+|Field |Type |Description
+
+|Title
+|Required
+|Title of the participatory process type. For instance, "Consultative".
+
```
Comment (andreslucena) [inline docs/en/modules/admin/pages/spaces/processes/types.adoc:26]: There's a missing closing table tag here. Without this the [HTML is broken](https://deploy-preview-138--decidim-documentation.netlify.app/en/develop/admin/spaces/processes/types):

Screenshot of the process types pages from netlify preview deployment

```suggestion

|===

```

**Review by andreslucena — COMMENTED** [review]
Two minor details to fix and we can merge and deploy this one

**Comment by RCheesley** [comment]
Thanks for the review @andreslucena - I've just committed the fixes proposed! Much appreciated, it's not easy wrapping your head around a new system and I figured someone who knew the quirks would be able to fix it in a few seconds!

**Review by andreslucena — COMMENTED** [review]
Great, thanks! I can't approve it as I've made some commits here, but this looks perfect!
I'll merge this.

**Pull request merged**